content/docs/command-reference/pull.md

```diff
@@ -107,89 +107,123 @@ reflinks or hardlinks to put it in the workspace without copying. See
 
 ## Examples
 
-For using the `dvc pull` command, a remote storage must be defined. (See
-`dvc remote add`.) For an existing <abbr>project</abbr>, remotes are usually
-already set up and you can use `dvc remote list` to check them. To remember how
-it's done, and set a context for the example, let's define a default SSH remote:
+Let's employ a simple <abbr>workspace</abbr> with some data, code, ML models,
+pipeline stages, such as the <abbr>DVC project</abbr> created for the
+[Get Started](/doc/tutorials/get-started). Then we can see what happens with
+`dvc pull`.
+
+<details>
+
+### Click and expand to setup the project
+
+Start by cloning our example repo if you don't already have it:
 
 ```dvc
-$ dvc remote add -d r1 ssh://_username_@_host_/path/to/dvc/remote/storage
-$ dvc remote list
-r1	ssh://_username_@_host_/path/to/dvc/remote/storage
+$ git clone https://github.com/iterative/example-get-started
+$ cd example-get-started
 ```
 
-> DVC supports several
-> [remote types](/doc/command-reference/remote/add#supported-storage-types).
+</details>
 
-Having some images and other files in remote storage, we can pull all changed
-files from the current Git branch:
+The workspace looks almost like in this
+[pipeline setup](/doc/tutorials/pipelines):
 
 ```dvc
-$ dvc pull --remote r1
+.
+├── data
+│   └── data.xml.dvc
+...
+└── train.dvc
 ```
 
-We can download specific files that are <abbr>outputs</abbr> of a specific
-DVC-file:
+We can now just run `dvc pull` to download the most recent `data/data.xml`,
+`model.pkl`, and other DVC-tracked files into the <abbr>workspace</abbr>:
 
 ```dvc
-$ dvc pull data.zip.dvc
+$ dvc pull
+
+$ tree example-get-started/
+example-get-started/
+├── data
+│   ├── data.xml
+│   ├── data.xml.dvc
+...
+├── model.pkl
+└── train.dvc
 ```
 
-In this case we left off the `--remote` option, so it will have pulled from the
-default remote. The only files considered in this case are what is listed in the
-`out` field of the DVC-file `targets`.
+We can download specific <abbr>outputs</abbr> of a single DVC-file:
+
+```dvc
+$ dvc pull train.dvc
+```
 
 ## Example: With dependencies
 
-Demonstrating the `--with-deps` option requires a larger example. First, assume
-a [pipeline](/doc/command-reference/pipeline) has been setup with these
+> If you followed previous example then delete the .dvc/cache directory with
+> `rm -Rf .dvc/cache`. Else `dvc status -c` would output
+> `Data and pipelines are up to date.`
```

Comment:
@jorgeorpinel The block quote for the 2nd example mentioned in the https://github.com/iterative/dvc.org/pull/1365#discussion_r432795831 is this.